OlehPavlyk/HW1/PT2/main.py

```diff
@@ -0,0 +1,35 @@
+print("choose the number of action")
```

Comment:
here is a shorter implementation:
a = int(input("Write your number, variable a ="))
b = int(input("Write your number, variable b ="))
print(f"""a + b = {a + b}
a - b = {a - b}
a * b = {a * b}
a / b = {a / b}
a**b = {a ** b}
a//b = {a // b}
a%b = {a % b}""")